admin-dev/themes/new-theme/js/pages/order/view.js

```diff
@@ -112,9 +112,8 @@ $(() => {
   function handlePrivateNoteChange() {
     const $submitBtn = $(OrderViewPageMap.privateNoteSubmitBtn);
 
-    $(OrderViewPageMap.privateNoteInput).on('input', (event) => {
-      const note = $(event.currentTarget).val();
-      $submitBtn.prop('disabled', !note);
+    $(OrderViewPageMap.privateNoteInput).on('input', () => {
+      $submitBtn.prop('disabled', false);
```

Comment:
So, if you remove the text and CTRL - Z you'r able to save it again ? Is that behavior wanted? We could add the content to a data-attr and compare the content on input but it's maybe too overkill, I don't know !